core/src/test/java/org/elasticsearch/action/support/replication/TransportReplicationActionTests.java

```diff
@@ -718,7 +718,7 @@ protected ReplicaResult shardOperationOnReplica(Request request, IndexShard repl
                 if (throwException) {
                     throw new ElasticsearchException("simulated");
                 }
-                return new ReplicaResult();
+                return new ReplicaResult(null);
```

Comment:
I expected to see some test that shows we respond with a failure if the (Primary|Replica)Result has a failure but I can't find it. Is there one?